app/models/conference.rb

```diff
@@ -15,4 +17,9 @@ def tickets_left
     confirmed_attendances = attendances.select { |attendance| attendance.confirmed }
     tickets.to_i - confirmed_attendances.size
   end
+
+  def chronology
+    return unless starts_on.present?
```

Comment:
I thought: the form is taking care of all this. In the form, a value will always be set (today's date). That is where the non-valid dates come from most of the time: when the start date is set in the future, the end date's default will still be today. 
Is it enough if the form handles this? Or shall I still add it to the model?